system/Commands/Generators/Views/model.tpl.php

```diff
@@ -0,0 +1,43 @@
+<@php
+
+namespace {namespace};
+
+use CodeIgniter\Model;
+
+class {class} extends Model
```

Comment:
If we're listing every option here then I would add recently introduced variable 
```php
protected $useAutoIncrement = true;
```

Reply:
Wow. I almost forgot that one! Thanks.